### PR TITLE
Add new localizations in english and spanish

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: web/static/i18n/en.json [hunk 1]
```diff
@@ -83,17 +83,17 @@
   "admin.email.pushServerEx": "E.g.: \"http://push-test.mattermost.com\"",
   "admin.email.testing": "Testing...",
   "admin.email.saving": "Saving Config...",
-  "admin.email.emailSuccess": "No errors were reported while sending an email.  Please check your inbox to make sure.",
+  "admin.email.emailSuccess": "No errors were reported while sending an email.Please check your inbox to make sure.",
```
Comment (jwilander) [inline web/static/i18n/en.json]: What happened to all the spaces after the period?

Comment (enahum) [inline web/static/i18n/en.json]: Wow right!! let me check that

**Comment by enahum** [comment]
fixed.

File: web/static/i18n/en.json [hunk 2]
```diff
@@ -1253,7 +1252,7 @@
   "intro_messages.teammate": "This is the start of your direct message history with this teammate. Direct messages and files shared here are not shown to people outside this area.",
   "intro_messages.offTopic": "<h4 class=\"channel-intro__title\">Beginning of {display_name}</h4><p class=\"channel-intro__content\">This is the start of {display_name}, a channel for non-work-related conversations.<br/></p>",
   "intro_messages.inviteOthers": "Invite others to this team",
-  "intro_messages.default": "<h4 class='channel-intro__title'>Beginning of {display_name}</h4><p class='channel-intro__content'><strong>Welcome to {display_name}!</strong><br/><br/>This is the first channel teammates see when they sign up - use it for posting updates everyone needs to know.</p>",
+  "intro_messages.default": "<h4 class='channel-intro__title'>Beginning of {display_name}</h4><p class='channel-intro__content'><strong>Welcome to {display_name}!'</strong><br/><br/>This is the first channel teammates see when they sign up - use it for posting updates everyone needs to know.</p>",
```
Comment (jwilander) [inline web/static/i18n/en.json]: Random `'` is getting added here

Comment (enahum) [inline web/static/i18n/en.json]: It's not random, it's on the code itself

```javascript
return (
        <div className='channel-intro'>
            <FormattedHTMLMessage
                id='intro_messages.default'
                defaultMessage="<h4 class='channel-intro__title'>Beginning of {display_name}</h4><p class='channel-intro__content'><strong>Welcome to {display_name}!'</strong><br/><br/>This is the first channel teammates see when they sign up - use it for posting updates everyone needs to know.</p>"
                values={{
                    display_name: channel.display_name
                }}
            />
            {inviteModalLink}
            {createSetHeaderButton(channel)}
            <br/>
        </div>
    );
```

I'm going to remove it from the code and from the loc file

**Comment by coreyhulen** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**